core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -654,6 +657,14 @@ portfolio.pending.step3_seller.onPaymentReceived.name=Please also verify that th
 portfolio.pending.step3_seller.onPaymentReceived.note=Please note, that as soon you have confirmed the receipt, the locked trade amount will be released to the BTC buyer and the security deposit will be refunded.
 portfolio.pending.step3_seller.onPaymentReceived.confirm.headline=Confirm that you have received the payment
 portfolio.pending.step3_seller.onPaymentReceived.confirm.yes=Yes, I have received the payment
+portfolio.pending.step3_seller.status.delayedPayoutInfo=Please open your Bisq application after {0} to release the Bitcoin.
+portfolio.pending.step3_seller.popup.delayedPayoutInfo=The buyer''s account has been created {0} ago.\n\n\
```

Comment:
```
portfolio.pending.step3_seller.popup.delayedPayoutInfo=The buyer''s account was created {0} ago.\n\n\
To reduce risks with bank chargebacks, the payout of the bitcoin is delayed until {1}.\n\n\
Please be sure to open your Bisq application after that date to release the bitcoin to the buyer, and to get your security deposit refunded.\n\n\
In case you get a chargeback request from your bank, open a dispute with Cmd + o or Ctrl + o.
```